src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -143,4 +148,35 @@ private Subscription getExistingSubscription(final SubscriptionBase subscription
                 subscriptionBase.getConsumerGroup());
     }
 
+    private final class EventTypesHolder {
```

Comment:
@adyach I thought you need two lists.
For me it better to do like that:
```json
List<String> result= types.stream().filter(type -> { 
try { 
    repository.findOne(type);
    return true;
} catch (Exception e) {
    return false;
}).collect(Collectors.toList());
```

```json
repository.exists(type)
``` 
could be just syntax sugar